tests/kernels/test_cutlass.py

```diff
@@ -87,61 +95,74 @@ def cutlass_int8_gemm_helper(m: int,
 @pytest.mark.parametrize("k", [128, 496, 1024])
 @pytest.mark.parametrize("per_act_token", [True, False])
 @pytest.mark.parametrize("per_out_ch", [True, False])
+@pytest.mark.parametrize("azp", [True, False])
 @pytest.mark.skipif(capability < 89,
                     reason="FP8 is not supported on this GPU type.")
 def test_cutlass_fp8_gemm(m: int, n: int, k: int, per_act_token: bool,
-                          per_out_ch: bool):
-    cutlass_fp8_gemm_helper(m, n, k, per_act_token, per_out_ch)
+                          per_out_ch: bool, azp: bool):
+    if azp and capability > 89:
+        pytest.skip()
```

Comment:
could you give it a reason? e.g. 
`pytest.skip(reason='bias is not yet supported by sm90 CUTLASS kernels')`

ditto for the other skips

Reply:
Will remove as we're going to add c3x in this pr anyway